stories/Molecules/Text/Authors/Authorcolumn/authors.scss

```diff
@@ -1,63 +1,99 @@
 @import '../../../../assets/scss/variables';
 @import '../../../../assets/scss/breakpoints';
 
-.author-wrapper {
-  border-top: 2px solid $color-black;
-  padding-top: $spacing-32;
-
-  &__box {
-    display: inline-flex;
-    margin-bottom: $spacing-24;
-    width: 100%;
-    
-    &.author-3-col {
-      margin-left: 0.5rem;
-      margin-right: 0.5rem;
-      width: calc(50% - 1rem);
+.author {
+  &-wrapper {
+    &--1 {
+      width: 100%;
+    }
+
+    &--2 {
+      @include devicebreak(medium) {
+        .author-wrapper__box {
+          width: calc(49% - 1.875rem);
   
-      .author__label {
-        padding: 0;
-        width: 100%;
-      }
+          &:nth-child(odd) {
+            margin-right: 1.875rem;
+          }
   
-      .author__img {
-        display: none;
+          &:nth-child(even) {
+            margin-left: 1.875rem;
+          }
+        }
       }
     }
+
+    &--3 {
+      @include devicebreak(medium) {
+        .author-wrapper__box {
+          width: calc(32% - 1.875rem);
```

Comment:
@pallavikadam-srijan please use the grid system and don't write the same custom styles for this

Reply:
done